### PR TITLE
move cu24, cu75, cu80/v1 into capsunlocked folder

### PR DESCRIPTION
**Pull request by peepeetee**
## Description

move cu24 and cu75 into capsunlocked folder

move keyboard/cu80 to keyboard/capsunlocked/cu80/v1

Also replaces readme images with imgur links

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by peepeetee** [comment]
@yiancar @aholland909

**Comment by yiancar** [comment]
My PCBs never got used for production.

**Comment by peepeetee** [comment]
> My PCBs never got used for production.

Could you elaborate?

File: keyboards/capsunlocked/cu80/v1/v1.c [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "v1.h"
```
Comment (drashna) [inline keyboards/capsunlocked/cu80/v1/v1.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

Comment (peepeetee) [reply keyboards/capsunlocked/cu80/v1/v1.c:18]: done

**Comment by drashna** [comment]
> My PCBs never got used for production.

Do you mean that one or more of the boards touched have no production models and can safely be removed?

**Comment by yiancar** [comment]
I think so BUT... only CapUnlocked themselves can answer that.

**Comment by peepeetee** [comment]
> I think so BUT... only CapUnlocked themselves can answer that.

How should we proceed? since they went insolvent, who should we ask? or should we just .... go forward with the current setup?

**Review by drashna — APPROVED** [review]
Well, for now, it's in the repo, so I'd say proceed with the assumption that there are units out there.  

It may be worth following up with CapsUnlocked to see if there are actual units, and remove them if there are not.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**